src/components/TypingGame.tsx

```diff
@@ -71,6 +71,10 @@ const TypingGame: React.FC = () => {
    */
   const handleKeyDown = useCallback((event: KeyboardEvent) => {
     if (gameState === 'paused' || gameState === 'end') {
+      // press space to start
+      if (event.key === '') {
```

Comment:
```suggestion
      if (event.code === 'Space') {
```
でお願いします！
(qwerty => dvrokeに変換するコード（`/lib/KeySwitcher.ts`）なんですが、現状タイピングに使用しないコードを打つとevent.keyとして`''`を返すようにしているので、スペース以外でもスタートしてしまう)